tsconfig.json

```diff
@@ -15,4 +43,9 @@
     "jsx": "react",
     "jsxFactory": "h",
   },
+  "exclude": ["packages/core/src/components/**/test/*", "**/*.d.ts", "**/*.stories.*", "**/*.axe.ts", "**/*.e2e.ts", "packages/core/src/stories/**", "packages/core/src/components/_beta/**"],
+  "include": ["packages/core/**/*"],
+  "files": [
+    "packages/core/src/global.d.ts"
+  ]
```

Comment:
I don't think stories and test files should be excluded from using strict mode. Ideally it should apply to all TS files in the whole repository.